### DISCUSSION TITLE
Loopback 3 Related Help

### DISCUSSION THREAD

**Discussion by kkretisio** [loopbackio/loopback-next#7597] [Q&A] [answered]
Hi Team,

We are using loopback 3 in our application with node version 8.x, We are planning to upgrade our node vision 14.x. in this case loopback3 will work without any issues?

Please suggest 
Thanks 
KK

**Comment by achrinza** [comment, marked as answer]
Written by @dhmlau from [Slack](https://loopbackio.slack.com/archives/C0114F6NMDJ/p1623427977007600?thread_ts=1623310641.005700&channel=C0114F6NMDJ&message_ts=1623427977.007600):

> looking at a PR in loopback repo. Node.js 14 is tested in loopback. https://github.com/strongloop/loopback/pull/4343
> ![image](https://user-images.githubusercontent.com/25147899/123156599-10656800-d49c-11eb-946a-ebecc8085346.jpeg)

Hope it helps!
